packages/voice-compass/src/index.ts

```diff
@@ -23,10 +23,21 @@ export interface Config {
 
 export interface StepData {
   stepId: string;
-  context?: Record<string, any>;
+  context?: Context;
 }
 
-export type Context = Record<string, any>;
+/** A JSON stringify-able value. */
+export type jsonable =
+  | string
+  | number
+  | boolean
+  | null
+  | undefined
+  | jsonable[]
+  | { [key: string]: jsonable }
+  | { toJSON(): jsonable };
+
+export type Context = Record<string, jsonable>;
```

Comment:
I'm not convinced this is really an improvement.

Reply:
mostly: functions

Reply:
I would stick to the `any` here, pretty sure some user will have a not-quite-so-perfectly typed codebase and the type definition will cause them trouble.

The other interesting avenue would be to add `context` as a kind of type parameter/generic to the client somehow, because this is code we can generate for download from the platform.